src/Data/Functor/Contravariant/Divisible.hs

```diff
@@ -611,8 +611,8 @@ instance Decidable SettableStateVar where
 -- @
 -- 'choose' 'Left' m ('lose' f)  = m
 -- 'choose' 'Right' ('lose' f) m = m
--- 'choose' f ('choose' g m n) o = 'divide' f' m ('divide' 'id' n o) where
---   f' bcd = 'either' ('either' 'id' ('Right' . 'Left') . g) ('Right' . 'Right') . f
+-- 'choose' f ('choose' g m n) o = 'choose' f' m ('choose' 'id' n o) where
+--   f' bcd = 'either' ('either' 'id' 'Left' . g) ('Right' . 'Right') . f
```

Comment:
Does that `bcd` argument to `f'` belong there? It looks like `f'` is supposed to be written pointfree (and with the way it's written right now, `f'` simply ignores `bcd`).

Reply:
Yes, I think you're right.

Reply:
OK. Remove `bcd`, and I'll merge :)

Reply:
Updated, sorry for the delay.

Maybe we should fix the line above too (#26)?

Reply:
If you have a suggestion for how to fix #26, I'd happily take it as a separate pull request.